microsoft/botframework-solutions
# [Skill Template][TypeScript] Implement Code Coverage

**DEPENDS ON #1361** 

## Description
- Implement `nyc` for **Code Coverage**
- Add `.nycrc` file
- Update `copier.js` to copy `nycrc` correctly
- Update `mocha.opts`
- Update `.gitignore`
- Update `sample-skill`

## Testing Steps

1. Go to `AI\templates\Virtual-Assistant-Template\typescript\samples\sample-skill`
2. Run `npm install`
3. Run `npm run coverage`
4. Check the following output after executing the command

![image](https://user-images.githubusercontent.com/11904023/57797494-f257a600-7720-11e9-9cd5-d466dfa5c129.png)
